tools/buildtools/ambertools/tleap.xml

```diff
@@ -0,0 +1,2043 @@
+            <param name="arg_solvateShell_arg_5" label="solvateShell_arg_5 value" type="text" value="" help=""/>
+       </when>
+       <when value="source">
+               <expand macro="builtin_amberfiles" />  
+       </when>
+       <when value="transform">
+            <param name="arg_atoms" label="atoms value" type="text" value="" help="CONTAINER/LIST"/>
+            <param name="arg_matrix" label="matrix value" type="text" value="" help="LIST"/>
+       </when>
+       <when value="translate">
+            <param name="arg_atoms" label="atoms value" type="text" value="" help="UNIT/RESIDUE/ATOM"/>
+            <param name="arg_direction" label="direction value" type="text" value="" help="LIST"/>
+       </when>
+       <when value="verbosity">
+            <param name="arg_level" label="level value" type="text" value="" help="NUMBER"/>
+       </when>
+       <when value="zMatrix">
+            <param name="arg_obj" label="obj value" type="text" value="" help="UNIT/RESIDUE/ATOM"/>
+            <param name="arg_zmatrix" label="zmatrix value" type="text" value="" help="LIST"/>
+       </when>
+    </conditional>
+</repeat>
+    </inputs>
+    <outputs>
+        <data format="txt" name="output_tleap_in" label="Build tLEaP: tleap.in"/>
+        <data format="txt" name="output_tleap" label="Build tLEaP: tleap.log"/>
+        <collection name="output_pdb" type="list" label="Collection of pdb" format="pdb">
+               <discover_datasets pattern="(?P&lt;designation&gt;.+)\.pdb" directory="out" format="pdb"/>
+        </collection>
+        <collection name="output_top" type="list" label="Collection of topology files" format="prmtop">
+               <discover_datasets pattern="(?P&lt;designation&gt;.+)\.prmtop" directory="out" format="prmtop"/>
+        </collection>
+        <collection name="output_txt" type="list" label="Collection of txt" format="txt">
+               <discover_datasets pattern="(?P&lt;designation&gt;.+)\.txt" directory="out" format="txt"/>
+        </collection>
+        <collection name="output_coord" type="list" label="Collection of coordinate files" format="inpcrd">
+               <discover_datasets pattern="(?P&lt;designation&gt;.+)\.inpcrd" directory="out" format="inpcrd"/>
+        </collection>
+        <collection name="output_mol2" type="list" label="Collection of mol2" format="mol2">
+               <discover_datasets pattern="(?P&lt;designation&gt;.+)\.mol2" directory="out" format="mol2"/>
+        </collection>
+        <collection name="output_mol3" type="list" label="Collection of mol3" format="mol3">
+               <discover_datasets pattern="(?P&lt;designation&gt;.+)\.mol3" directory="out" format="mol3"/>
+        </collection>
+    </outputs>
+    <tests>
+        <test>
+          <repeat name="tleap_cmds">
+            <conditional name="tleap_cond">
+                 <param name="tleap_cmd" value="source"/>
+                   <param name="arg_filename" value="oldff/leaprc.ff14SB"/>
```

Comment:
indentation is off here ... do you know about https://github.com/galaxyproject/galaxy-language-server ?

It will help you with style but also with macros etc ...

Reply:
corrected

Reply:
thanks for the info! installed vscode extension, should help going forward